python/ql/lib/change-notes/2023-08-07-serverless-sources.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* Added remote flow sources for serverless functions. These are automatically included in the `RemoteFlowSource` class.
```

Comment:
```suggestion
* Added modeling of AWS Lambda handlers that can be identified with `AWS::Serverless::Function` in YAML files, where the event parameter is modeled as a remote-flow-source.
```

Reply:
Not sure why this is better. Hopefully the model is a bit more general than AWS (even though connexion seems to use a different scheme). Is calling out the event parameter useful in this context?

Reply:
> Added remote flow sources for serverless functions.

Of the 3 major cloud providers we now have support for AWS Lambda (at least some), but no support at all for Google Cloud Functions or Azure Functions. So claiming we have support for serverless functions is a bit too wide, and we need to be more precise. Since I saw a need to be more specific, I just mentioned AWS since that's the key part of this PR (I know that we technically support more than AWS, so feel free to mention that as well if you feel it's important... I just didn't)

Furthermore, it's possible to configure your AWS Lambda in the AWS console, such that a YAML file is never used, and we therefore can't identify the actual AWS Lambda handler. (neither can we figure it out if code is inlined by using `AWS::Lambda::Function`).

Reply:
Thanks for elaborating :-)